### ISSUE TITLE
Simple usage chaning type of error. Minimal working example inside

### ISSUE THREAD

**Issue by RusAlex** [squizlabs/PHP_CodeSniffer#860]
I've just starting using PHPCS project. And want to set up rules for my ongoing project. 

I set up my own standard using wiki of the project. But I have a probem, I cant convert error to types and cant amend messages via ruleset.xml 

I've prepared a minimal working example repository.

Here is my config:

```xml
<?xml version="1.0"?>
<ruleset name="CodingStandard">
 <description>The MySource coding standard builds on the Squiz coding standard. Currently used for MySource Mini development.</description>

 <rule ref="PEAR.NamingConventions.ValidVariableName">
   <type>warning</type>
   <message>test</message>
 </rule>

</ruleset>
```

And here is my repository where error can be reproduced.

https://github.com/RusAlex/phpcs-mwe

**Comment by aik099** [comment]
In above example `PEAR.NamingConventions.ValidVariableName` is sniff name, but you need to address a specific error code, that sniff produces, like `PEAR.NamingConventions.ValidVariableName.PublicUnderscore` for example.

Actual error codes can be seen, when you add `-s` to the `phpcs` command.

**Comment by RusAlex** [comment]
You saved my day. Thanks you very much.

**Comment by aik099** [comment]
Good. If described problem is no longer relevant, then please feel free to close the task.
